app/views/api-docs/examples/index.html.md

```diff
@@ -0,0 +1,191 @@
+# e-Manifest API Examples | US EPA
+
+Code examples of using the e-Manifest API.
+
+Using the e-Manifest API consists of two major steps: uploading and signing manifests.
+
+The production base URL is **https://e-manifest.18f.gov/api/v0**
+
+Notation note: path variables are expressed as `:variable_name`. The colon is not part of the URI.
+
+## <a name="submit-manifest"></a>Submit a manifest
+
+```bash
+curl -i -X POST -H 'Content-Type: application/json' \
+  --data @manifest.json \
+  https://e-manifest.18f.gov/api/v0/manifests?tracking_number=:manifest_tracking_number
```

Comment:
I think this example is no longer accurate. should just be `https://e-manifest.18f.gov/api/v0/manifests` with params for the `content` values